### ISSUE TITLE
Improvement - mark comments as completed

### ISSUE THREAD

**Issue by frejfrej** [reywood/bitbucket-pr-review#2]
When reviewing, you can have some conversations that remain in the PR because the related update is not on the specific line you commented on, or it is on the file itself.

It would be nice to have the ability to mark a conversation as done / completed.

This would hide the comment and add a small (X) label (X being the number of hidden comments in the hidden conversation). Clicking the (X) would restore / unhide the conversation.

**Comment by frejfrej** [comment]
In fact, having all comments marked "completed" could be considered a requirement before clicking "Approve"...

**Comment by reywood** [comment]
Good idea! I usually use tasks to denote when a comment has been addressed, but I can definitely see how collapsing comments would be useful. Can't promise when, but I'll look into this when I get a chance.
